framework/src/modules/chain/submodules/blocks/chain.js

```diff
@@ -837,14 +837,9 @@ Chain.prototype.deleteLastBlock = function(cb) {
 				});
 			},
 			updateApplicationState(seriesCb) {
-				return library.storage.entities.Block.get(
-					{},
-					{
-						limit: 5,
-						sort: 'height:desc',
-					}
-				)
-					.then(blocks => {
+				return modules.blocks
+					.calculateNewBroadhash()
+					.then(newState => {
```

Comment:
We are calling `.calculateNewBroadhash()` and then using variable name `newState`, which leads to understanding that first method is returning a state, which is not true. We may change it to 

```
then ({broadhash, height} => {
```
That will make clear what first method is returning.